pkg/network/setup/network_test.go

```diff
@@ -288,6 +300,85 @@ var _ = Describe("VMNetworkConfigurator", func() {
 			})
 		})
 	})
+
+	Context("pod network phase1", func() {
+		var (
+			mockNetworkH *netdriver.MockNetworkHandler
+		)
+
+		BeforeEach(func() {
+			ctrl := gomock.NewController(GinkgoT())
+			mockNetworkH = netdriver.NewMockNetworkHandler(ctrl)
+
+			dutils.MockDefaultOwnershipManager()
+		})
+
+		It("fails setup during network discovery", func() {
+			vmi := newVMIBridgeInterface("testnamespace", "testVmName")
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
+			vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+			vmNetworkConfigurator := newVMNetworkConfiguratorWithHandlerAndCache(vmi, mockNetworkH, &baseCacheCreator)
```

Comment:
These four lines repeat themselves three times, please consider moving them to `BeforeEach`.

Reply:
I tried to avoid the definition of variables between the closures.
I am not sure which option is better, so I will just change it per your suggestion.

Done